lnbits/wallets/phoenixd.py

```diff
@@ -216,7 +216,7 @@ async def paid_invoices_stream(self) -> AsyncGenerator[str, None]:
                     while settings.lnbits_running:
                         message = await ws.recv()
                         message_json = json.loads(message)
-                        if message_json and message_json["type"] == "payment-received":
+                        if message_json and message_json["type"] == "payment_received":
```

Comment:
Maybe do this to be more defensive?

```suggestion
                        if message_json and message_json.get("type") == "payment_received":
```

Reply:
@prusnak looks good. 

Are we actually using the paid_invoices_stream? Or this is a future thing?

Reply:
Yes it is being used, that is how you are notified that one of your invoices has been paid.